cinema/views.py

```diff
@@ -1 +1,63 @@
-# write views here
+from rest_framework.viewsets import ModelViewSet
+from cinema.models import Actor, CinemaHall, Genre, Movie, MovieSession
+from cinema.serializers import (
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports) 